internal/speedtest/runner/runner.go

```
@@ -0,0 +1,751 @@
+			}
+			res.Rounds = append(res.Rounds, round)
+			res.Degraded = true
+			return
+		}
+
+		xfer := transfer.Run(ctx, client, spec, dir, threads, time.Duration(cfg.TimeoutMs)*time.Millisecond,
+			func(dir transfer.Direction, totalBytes int64, elapsed time.Duration, mbps float64) {
+				if bus == nil {
+					return
+				}
+				bus.Progress(speedtest.Text(cfg.Language, strings.Title(string(dir)), mapDirectionZH(dir)),
```

golangci-lint (2.11.4)

[error] SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. 

(staticcheck)